### ISSUE TITLE
step() missing 1 required positional argument: 'metrics'

### ISSUE THREAD

**Issue by AceGuoX** [PaddlePaddle/Paddle#40436]
为使您的问题得到快速解决，在建立Issues前，请您先通过如下方式搜索是否有相似问题:【搜索issue关键字】【使用labels筛选】【官方文档】

如果您没有查询到相似问题，为快速解决您的提问，建立issue时请提供如下细节信息：
- 标题：简洁、精准概括您的问题，例如“Insufficient Memory xxx" ”
- 版本、环境信息：
    1）PaddlePaddle版本：2.2.2
    4）系统环境：paddle官方平台
- 训练信息
![图片](https://user-images.githubusercontent.com/38156085/157797221-4493b690-21b7-4b7c-b0a1-495c5b03168d.png)

- 复现信息：
````
from paddle.vision.transforms import ToTensor
class Mnist(paddle.nn.Layer):
    def __init__(self):
        super(Mnist, self).__init__()
        self.flatten = paddle.nn.Flatten()
        self.linear_1 = paddle.nn.Linear(784, 512)
        self.linear_2 = paddle.nn.Linear(512, 10)
        self.relu = paddle.nn.ReLU()
        self.dropout = paddle.nn.Dropout(0.2)

    def forward(self, inputs):
        y = self.flatten(inputs)
        y = self.linear_1(y)
        y = self.relu(y)
        y = self.dropout(y)
        y = self.linear_2(y)
        return y

train_dataset =paddle.vision.datasets.MNIST(mode='train', transform=ToTensor())
mnist = Mnist()
inputs = paddle.static.InputSpec([-1, 1, 28, 28], dtype='float32', name='input')
label = paddle.static.InputSpec([-1, 1], dtype='int8', name='label')
model = paddle.Model(mnist, inputs, label)
learning_rate = paddle.optimizer.lr.ReduceOnPlateau(learning_rate=0.05, factor=0.01, patience=5, verbose=True)
model.prepare(paddle.optimizer.Adam(parameters=model.parameters(),learning_rate = learning_rate), 
              paddle.nn.CrossEntropyLoss(),
              paddle.metric.Accuracy())
model.fit(train_dataset, 
          epochs=5, 
          batch_size=64,
          verbose=1)
````
- 问题描述：使用高层API进行训练时，使用ReduceOnPlateau自适应的学习率衰减策略报错，改用PiecewiseDecay策略能正常使用

**Comment by zhwesky2010** [comment]
@AceGuoX 这个学习率策略和其他不同，step时需要输入loss，`scheduler.step(loss)`，需要在高层API `model.fit` 里进行支持，基础API无需调整

**Comment by AceGuoX** [comment]
> @AceGuoX 这个学习率策略和其他不同，step时需要输入loss，`scheduler.step(loss)`，需要在高层API `model.fit` 里进行支持，基础API无需调整

就是说现在高层API`model.fit`不能用那个学习策略是嘛

**Comment by qingqing01** [comment]
现在的确model.fit不支持和ReduceOnPlateau搭配使用
